analysed-packages/imx-gpu-viv/version-6.4.3.p4.6/README.md

```diff
@@ -0,0 +1,17 @@
+## Download Location
+
+https://github.com/Freescale/kernel-module-imx-gpu-viv/archive/refs/tags/upstream/6.4.3.p4.6.zip
+
+## Package URL (purl)
+
+pkg:generic/imx-gpu-viv@6.4.3.p4.6?download_url=https://github.com/Freescale/kernel-module-imx-gpu-viv/archive/refs/tags/upstream/6.4.3.p4.6.zip
```

Comment:
This is strange, I downloaded the package from the link which is given here. I analysed this package I downloaded: all the files which carry a licensed information (this are 107, except the text of the GPL-2.0) have a dual license statement in it GPL-2.0-or-later or MIT. But in your analysis there are much more licenses concluded. The source package I downloaded did not contain any of such licenses especially no commercial license. 
So the package you have analysed was probably not downloaded from the given link.

Reply:
@dierberg I downloaded the binary blob, it contains a lot of shared objects and header files. I did not find da header file which is dual licensed. Whereas the Freescale package is dual licensed. In short I am really puzzled. I would like to work on the PR but I do not know what I can do, this is at lieast from the information I have inconsistent.
Besides that - I think that trying to do license and copyright analysis of binaries (.so) will not lead to a complete picture of it.